plugins/inputs/lustre2/lustre2.go

```diff
@@ -376,8 +379,41 @@ func (*Lustre2) SampleConfig() string {
 	return sampleConfig
 }
 
+func (l *Lustre2) GetLustreHealth() error {
+	filename := filepath.Join(l.rootdir, "/sys", "fs", "lustre", "health_check")
+	if _, err := os.Stat(filename); err != nil {
+		// try falling back to the old procfs location
+		// it was moved in https://github.com/lustre/lustre-release/commit/5d368bd0b2
+		filename = filepath.Join(l.rootdir, "/proc", "fs", "lustre", "health_check")
```

Comment:
My bad on the regression here. As this is a linux only plugin do we want to reduce this down to joining the two strings or we can call filepath.Abs on the result.

Reply:
Yeah I think that's a pretty good solution. Thanks for the tip, I was just sitting down to look at this! My original implementation is technically a tiny bit more flexible in that it would work for a relative rootdir. But I don't see any reason we'd ever need to support that and I like how your approach doesn't need a `nolint`.

Reply:
Actually that didn't work either. `filepath.Abs()` inserts the CWD (e.g. `/tmp/`), not `/`. I hope my new way to fix it will make the linter happy ...